fuzzers/utils.py

```diff
@@ -108,7 +108,7 @@ def restore_directory(directory):
     initial_cwd = os.getcwd()
     with tempfile.TemporaryDirectory() as temp_dir:
         backup = os.path.join(temp_dir, os.path.basename(directory))
-        shutil.copytree(directory, backup)
+        shutil.copytree(directory, backup, symlinks=True)
```

Comment:
Where are symlinks used ?

Reply:
The AFLPlusPlus fuzzers have multiple symlinks which point to the same binary, so they fail when the symlinks are called and copytree doesn't copy them.